lib/safe-pg-migrations/plugins/idem_potent_statements.rb

```diff
@@ -11,52 +12,62 @@ def add_index(table_name, column_name, **options)
       remove_index(table_name, name: index_name)
       super
     end
+    ruby2_keywords :add_index if respond_to?(:ruby2_keywords, true)
 
-    def add_column(table_name, column_name, type, options = {})
+    def add_column(table_name, column_name, type, *)
       return super unless column_exists?(table_name, column_name)
 
       SafePgMigrations.say("/!\\ Column '#{column_name}' already exists in '#{table_name}'. Skipping statement.", true)
     end
+    ruby2_keywords :add_column if respond_to?(:ruby2_keywords, true)
 
-    def remove_column(table_name, column_name, type = nil, options = {})
+    def remove_column(table_name, column_name, type = nil, *)
       return super if column_exists?(table_name, column_name)
 
       SafePgMigrations.say("/!\\ Column '#{column_name}' not found on table '#{table_name}'. Skipping statement.", true)
     end
+    ruby2_keywords :remove_column if respond_to?(:ruby2_keywords, true)
 
-    def remove_index(table_name, options = {})
+    def remove_index(table_name, *args)
+      options = args.last.is_a?(Hash) ? args.last : {}
       index_name = options.key?(:name) ? options[:name].to_s : index_name(table_name, options)
 
       return super if index_name_exists?(table_name, index_name)
 
       SafePgMigrations.say("/!\\ Index '#{index_name}' not found on table '#{table_name}'. Skipping statement.", true)
     end
+    ruby2_keywords :remove_index if respond_to?(:ruby2_keywords, true)
 
-    def add_foreign_key(from_table, to_table, **options)
-      options_or_to_table = options.slice(:name, :column).presence || to_table
-      return super unless foreign_key_exists?(from_table, options_or_to_table)
+    def add_foreign_key(from_table, to_table, *args)
+      options = args.last.is_a?(Hash) ? args.last : {}
+      suboptions = options.slice(:name, :column)
+      return super unless foreign_key_exists?(from_table, suboptions.present? ? nil : to_table, **suboptions)
 
       SafePgMigrations.say(
         "/!\\ Foreign key '#{from_table}' -> '#{to_table}' already exists. Skipping statement.",
         true
       )
     end
+    ruby2_keywords :add_foreign_key if respond_to?(:ruby2_keywords, true)
 
-    def create_table(table_name, comment: nil, **options)
+    def create_table(table_name, *args)
```

Comment:
`def create_table(table_name, id: :primary_key, primary_key: nil, force: nil, **options)`

Reply:
I'm not sure for that one, we would need to pass `id`, `primary_key` and `force` by hand on the line 60.